EazyShop/ESNav.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ESNav.swift
+//  EazyShop
+//
+//  Created by Cesar Rojas on 7/31/24.
+//
+
+import SwiftUI
+
+struct ESNav: View {
+    @State private var navigateToSignUp: Bool = false
+    
+    var body: some View {
+        NavigationView {
+            HStack {
+                Button("Login Page") {
```

Comment:
Create a constants file for All the titles, or strings you are using in the app, so this way you can update at one place and it will reflect throughout the app

Reply:
Constants file has been created to manage all the Titles, Text, Buttons Text, Fonts, Colors and Images strings.